functionary/ui/views/scheduling.py

```diff
@@ -269,14 +275,18 @@ def _create_scheduled_task(
 ):
     """Helper function for creating scheduled task"""
     with transaction.atomic():
-        scheduled_task = ScheduledTask.objects.create(
+        scheduled_task = ScheduledTask(
```

Comment:
This is a case where using `objects.create()` or skipping the `clean()` is okay.  You are basically saying as much in your comment below because you are expecting the data to come in pre-validated.  As is, you're validating the data twice without any changes having happened in between.

The validation isn't especially expensive in this case and I suppose it does protect you from a scenario where someone messes up and passes non-validated data into this function, but given that it's an internal helper, I think trusting it and just noting in the docstring that the data is assumed to have be pre-validated would be acceptable.

Reply:
Ok, makes sense. I wasn't sure if we wanted to just switch to the object, clean, save flow for everything, so I had included that comment. Fixed